app/src/main/java/com/google/samples/apps/sunflower/GardenActivity.kt

```diff
@@ -30,20 +28,19 @@ import com.google.samples.apps.sunflower.databinding.ActivityGardenBinding
 
 class GardenActivity : AppCompatActivity() {
 
-    private lateinit var drawerLayout: DrawerLayout
+    private lateinit var binding: ActivityGardenBinding
```

Comment:
Let's keep this as `drawerLayout` - see comment in `onBackPressed`.

Reply:
ok, I've reset it.